image/image_test.go

```diff
@@ -2584,7 +2585,7 @@ func (s *imageSuite) TestSetupSeedCore20(c *C) {
 	s.makeSnap(c, "pc-kernel=20", nil, snap.R(1), "")
 	gadgetContent := [][]string{
 		{"grub-recovery.conf", "# recovery grub.cfg"},
```

Comment:
also what to do in the test about grub-recovery.conf? this content is no longer installed, so should we assert that there is not a grub-recovery.conf in the resultant image dirs ?

Reply:
I've intentionally left the file around as the previous code would copy its contents to grub.cfg. Instead of checking whether a file does not exist in the target directory, I've tweaked the test to verify that grub.cfg is the only file there.